recipes/aws-iam-authenticator/download-and-calculate-sha256.bash

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+version="1.11.5"
```

Comment:
I think you don't need this file right!?

Reply:
No, this is mostly an utility.

Reply:
Do you think I should just remove it?

Reply:
I think would be better if you remove it, if you don't mind